packages/calcite-components/src/components/combobox/combobox.tsx

```diff
@@ -1103,7 +1105,12 @@ export class Combobox
         }
       });
 
+      this.filterTextMatchPattern = this.filterText && new RegExp(`(${this.filterText})`, "i");
+
       this.filteredItems = this.getFilteredItems();
+      this.filteredItems.forEach((item) => {
+        item.filterTextMatchPattern = this.filterTextMatchPattern;
```

Comment:
It feels a little odd passing a regex property to children. It seems "safer" to pass just the filterText but I do get creating the regex on each instance might not be as performant.

Reply:
Not sure why this would be odd. We pass parent-level props to children in other components and we only need to build the pattern at the root.

Reply:
Yeah, its fine. Just stuck out to me. Proceed!